docs/userGuide/templates.md

```diff
@@ -0,0 +1,22 @@
+<variable name="title" id="title">Templates</variable>
+<frontmatter>
+  title: "User Guide: {{ title }}"
+  layout: userGuide
+</frontmatter>
+
+# Templates
+
+Markbind allows you to start off your new project with different templates.
+
+During initialization you can add a flag `--template <template-key>` to select a different template to initialize with. For example:
+
+```
+markbind init --template minimal
+```
+
+## Supported Templates
+
+Name    | Template key | Description
+----    | -------      | ------
+Default | `default`    | Default template filled with Markbind features to guide you to author better content
```

Comment:
Can mention in the description that "(this is the template used by default if `--template` is not specified)"